turbine/benches/cluster_nodes.rs

```diff
@@ -35,19 +37,27 @@ fn get_retransmit_peers_deterministic(
     slot_leader: &Pubkey,
     num_simulated_shreds: usize,
 ) {
-    let parent_offset = u16::from(slot != 0);
-    for i in 0..num_simulated_shreds {
-        let index = i as u32;
-        let shred = Shred::new_from_data(
-            slot,
-            index,
-            parent_offset,
-            &[],
-            ShredFlags::empty(),
-            0,
-            0,
-            0,
-        );
+    let keypair = Keypair::new();
+    let merkle_root = Some(Hash::default());
+    let reed_solomon_cache = ReedSolomonCache::default();
+    let mut stats = ProcessShredsStats::default();
+    let parent_slot = if slot > 0 { slot - 1 } else { 0 };
+    let shredder = Shredder::new(slot, parent_slot, 0, 0).unwrap();
+
+    let shreds = shredder
+        .make_merkle_shreds_from_entries(
```

Comment:
The way you are calling this it will always make exactly 64 shreds. you should either make sure the number of produced shreds is what num_simulated_shreds calls for, or remove the num_simulated_shreds completely (as it is const anyway and not critical to the purpose of the bench)